src/allmydata/interfaces.py

```diff
@@ -105,7 +105,7 @@ def advise_corrupt_share(reason=bytes):
 
 
 class RIStorageServer(RemoteInterface):
-    __remote_name__ = b"RIStorageServer.tahoe.allmydata.com"
+    __remote_name__ = native_str("RIStorageServer.tahoe.allmydata.com")
```

Comment:
if we know why foolscap needs a native string here, a comment would be great :)

Reply:
Uhhh I guess I can try to find out :)

Reply:
Probably because of this line: https://github.com/warner/foolscap/blob/43f4485a42c9c28e2c79d655b3a9e24d4e6360ca/src/foolscap/remoteinterface.py#L67